### ISSUE TITLE
screenreader test failing

### ISSUE THREAD

**Issue by mfranzke** [db-ui/mono#2921]
### Which generators are impacted?

- [X] All
- [ ] HTML
- [ ] React
- [ ] Angular
- [ ] Vue
- [ ] Web components
- [ ] Power Apps

### Reproduction case

- create a PR with a totally unrelated change to our component code, like e.g. https://github.com/db-ui/mono/pull/2911
- You'll recognize a failing job https://github.com/db-ui/mono/actions/runs/10100872256/job/27933559239?pr=2911

### Expected Behaviour

The pipeline for QS is running through without a failure.

### Browser version

None

**Comment by mfranzke** [comment]
seems to have been a temporary glitch
